tests/kafkatest/services/performance/end_to_end_latency.py

```diff
@@ -74,12 +74,15 @@ def __init__(self, context, num_nodes, kafka, topic, num_records, compression_ty
     def start_cmd(self, node):
         args = self.args.copy()
         args.update({
-            'zk_connect': self.kafka.zk_connect_setting(),
             'bootstrap_servers': self.kafka.bootstrap_servers(self.security_config.security_protocol),
             'config_file': EndToEndLatencyService.CONFIG_FILE,
             'kafka_run_class': self.path.script("kafka-run-class.sh", node),
             'java_class_name': self.java_class_name()
         })
+        if node.version < V_0_9_0_0:
```

Comment:
It would be good to have a function in `version.py` to reflect this

Reply:
> would be good to have a function in version.py to reflect [functionality unsupported below 0.9.0]

I'll add this function:

```
    def supports_bootstrap_server(self):
        return self >= V_0_9_0_0
```

I then made the changes all over since this 0.9.0 constant is checked in several places:

```
tests/kafkatest/services/console_consumer.py
tests/kafkatest/services/performance/consumer_performance.py
tests/kafkatest/services/performance/end_to_end_latency.py
tests/kafkatest/services/performance/producer_performance.py
tests/kafkatest/tests/core/upgrade_test.py
tests/kafkatest/tests/core/upgrade_test.py
```

We can revert if you think I went too far.

Reply:
thanks.  that sounds good!

can you name it something like `consumer_supports_bootstrap_server`?  There are a bunch of other bootstrap server functions (`acl_command_supports_bootstrap_server`, `topic_command_supports_bootstrap_server`, etc.) so it would be good to be clear

Reply:
> name it something like consumer_supports_bootstrap_server... bunch of other bootstrap server functions

Good point.  I renamed it `consumer_supports_bootstrap_server()` as you suggested.